README.md

```diff
@@ -30,8 +30,8 @@ is supported. Mash is configured to never return sequences with a distance great
 
 ## Requirements
 
-Java 8 (OpenJDK OK)  
-MongoDB 2.6+ (https://www.mongodb.com/)  
+Java 8 (OpenJDK OK)
+MongoDB 7.0.4+ (https://www.mongodb.com/)
```

Comment:
Oh, the spaces after these lines are necessary - 2 spaces = newline in markdown

Reply:
fixed